tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "allowJs": true,
     "baseUrl": ".",
     "skipLibCheck": true,
-    "strict": false,
+    "strictNullChecks": true,
```

Comment:
Why did we change this? This change causes the build error. We shouldn't be touching the tsconfig in the PR as far as I'm aware.

As an aside, 🍹, we should be using `"strict": true` which enforces `strictNullChecks` along with some other ones. It ensures type safety throughout the codebase.